cfme/web_ui/__init__.py

```diff
@@ -3399,7 +3399,8 @@ def get_cell(self, row_text, col_index):
             Selenium element of the cell.
         """
         self.expand_all_sections()
-        cell_loc = ".//div/div[1][contains(., '{}')]/../div[{}]".format(row_text, col_index + 2)
+        cell_loc = ".//div/div[1][contains(., '{}')]//tbody/tr[{}]/td".format(
```

Comment:
For just debugging purpose i also tried this one:

`cell_loc = "//div[contains(normalize-space(.), '{}')]/table/tbody/tr[12]/td[{}]".format(row_text, col_index + 1))`
getting the same result `SoftAssertionError`